docs/python/ReadMeOV.rst

```diff
@@ -0,0 +1,79 @@
+OpenVINO™ Execution Provider for ONNX Runtime
+===============================================
+
+-  `OpenVINO™ Execution Provider for ONNX Runtime <https://github.com/microsoft/onnxruntime/>`_. is a product designed for OnnxRuntime developers who want to get started with OnnxRuntime in their inferencing applications. This product delivers  `OpenVINO™ <https://software.intel.com/content/www/us/en/develop/tools/openvino-toolkit.html>`_. inline optimizations which enhance inferencing performance with minimal code modifications. OpenVINO™ Execution Provider for ONNX Runtime accelerates inference across many  `AI models <https://github.com/onnx/models>`_. on a variety of Intel silicon such as:
```

Comment:
1. Remove the dot '.' at the end of all the links in these sentences.
2. Shouldn't "get started with OnnxRuntime" be "get started with OpenVINO"?